jans-cedarling/cedarling/src/common/policy_store/mock_jwks_server.rs

```diff
@@ -0,0 +1,503 @@
+            .with_status(200)
+            .with_header("content-type", "application/json")
+            .with_body(
+                json!({
+                    "keys": [key_pair.jwk.clone()]
+                })
+                .to_string(),
+            )
+            .expect_at_least(0)
+            .create();
+
+        Ok(Self {
+            server,
+            key_pair,
+            endpoints: MockEndpoints { oidc_config, jwks },
+        })
+    }
+
+    /// Get the base URL of the mock server.
+    pub fn url(&self) -> String {
+        self.server.url()
+    }
+
+    /// Get the OpenID Configuration endpoint URL.
+    pub fn oidc_config_url(&self) -> String {
+        format!("{}{}", self.server.url(), Self::OIDC_CONFIG_PATH)
+    }
+
+    /// Get the JWKS endpoint URL.
+    pub fn jwks_url(&self) -> String {
+        format!("{}{}", self.server.url(), Self::JWKS_PATH)
+    }
+
+    /// Generate a trusted issuer configuration for use in policy stores.
+    pub fn trusted_issuer_json(&self, issuer_id: &str) -> Value {
+        json!({
+            issuer_id: {
+                "name": format!("Mock Issuer {}", issuer_id),
+                "oidc_endpoint": self.oidc_config_url(),
+                "token_metadata": {
+                    "access_token": {
+                        "user_id": "sub",
+                        "required_claims": ["sub", "aud"]
+                    },
+                    "id_token": {
+                        "user_id": "sub",
+                        "required_claims": ["sub", "email"]
+                    }
+                }
+            }
+        })
+    }
+
+    /// Generate a trusted issuer configuration with custom token metadata.
+    pub fn trusted_issuer_json_with_metadata(
+        &self,
+        issuer_id: &str,
+        token_metadata: Value,
+    ) -> Value {
+        json!({
+            issuer_id: {
+                "name": format!("Mock Issuer {}", issuer_id),
+                "oidc_endpoint": self.oidc_config_url(),
+                "token_metadata": token_metadata
+            }
+        })
+    }
+
+    /// Generate a test token with the given claims.
+    ///
+    /// Automatically adds `iss` claim set to the server URL.
+    pub fn generate_token(&self, mut claims: Value) -> Result<String, MockServerError> {
+        claims["iss"] = json!(self.url());
+        self.key_pair.sign_token(&claims)
+    }
+
+    /// Generate a token with standard claims.
+    ///
+    /// # Arguments
+    /// * `sub` - Subject (user ID)
+    /// * `aud` - Audience
+    /// * `extra_claims` - Additional claims to merge
+    pub fn generate_standard_token(
+        &self,
+        sub: &str,
+        aud: &str,
+        extra_claims: Option<Value>,
+    ) -> Result<String, MockServerError> {
+        use std::time::{SystemTime, UNIX_EPOCH};
+
+        let now = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap_or_default()
+            .as_secs();
```

Comment:
We can use chrono.
```suggestion
        let now = chrono::Utc::now().timestamp();
```

Can you add `std::time::SystemTime::now` as forbidden/disabled to `clippy.toml`? Using that method may lead to error, because may work not correctly using WASM.